### ISSUE TITLE
does it support running in docker ?

### ISSUE THREAD

**Issue by rayshaw001** [srlabs/extractor#5]
does it support running in docker ?

**Comment by rayshaw001** [comment]
I build a docker image with this tool,
and run this command in docker
`./extrator.py`
then I got blew error
`Traceback (most recent call last):
  File "/home/work/extractor/./extractor.py", line 37, in <module>
    from lxml import etree
ImportError: /usr/local/lib/python3.10/dist-packages/lxml/etree.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyGen_Send`

**Comment by stze** [comment]
Yes, you can follow those [instructions](https://github.com/srlabs/extractor#docker)
